### PR TITLE
CEDS-2027 movement details

### PR DESCRIPTION
**Pull request by MaciejRewera**

### CODE REVIEW

File: app/views/components/gds/inputDate.scala.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import forms.common.Date.{dayKey, monthKey, yearKey}
+
+@this(dateInput: GovukDateInput)
+
+@(
+  field: Field,
```
Comment (ed0906) [inline app/views/components/gds/inputDate.scala.html]: Looks like you just use this to get `field.name` maybe just pass that in instead?

File: app/views/components/gds/inputTime.scala.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import forms.common.Time.{hourKey, minuteKey}
+
+@this(dateInput: GovukDateInput)
+
+@(
+  field: Field,
```
Comment (ed0906) [inline app/views/components/gds/inputTime.scala.html]: Similar, looks like you just use this to get `field.name`. Maybe just pass that in instead?

**Comment by MaciejRewera** [comment]
I've addressed you comment. This was mostly copied from the previous implementation. But it turned out to be better after changed proposed.
Had to force-push as I've removed a commit from thte history.

**Comment by MaciejRewera** [comment]
Another force-push, as I did not rebase.

**Pull request merged**